src/hearings/containers/request-hearing/location-search-container/location-search-container.component.ts

```diff
@@ -1,16 +1,29 @@
 import { Component, OnInit } from '@angular/core';
-import { select, Store } from '@ngrx/store';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { Store, select } from '@ngrx/store';
+import { LocationModel } from 'api/locations/models/location.model';
+import { Observable, of } from 'rxjs';
 import { map } from 'rxjs/operators';
 import * as fromHearingStore from '../../../../hearings/store';
-
 @Component({
   selector: 'exui-location-search-container',
   templateUrl: './location-search-container.component.html',
+  styleUrls: ['./location-search.component.scss']
 })
 export class LocationSearchContainerComponent implements OnInit {
+  public locationType: string;
+  public findLocationFormGroup: FormGroup;
+  public selectedLocations$: Observable<LocationModel[]>;
+  public selectedLocation: LocationModel;
```

Comment:
```suggestion
  public selectedLocation: LocationByEPIMSModel ;
```

Reply:
Hi @rayliangatos this has been done

Reply:
what I mean here is EMPTY_SPACE before ;